File: 241230/a 부터 b 까지/a-to-b.py
a, b = map(int, input().split())

print(a, end=" ")

while a<=b:
    if a%2==1:
        a*=2
        if a <= b:
            print(a, end=" ")
    else:
        a+=3
        if a <= b:
            print(a, end=" ")

File: 250701/A 부터 B 까지/a-to-b.js
const fs = require('fs');
let input = fs.readFileSync(0).toString().trim().split(" ");
let A = Number(input[0]);
let B = Number(input[1]);
let result = "";
while (A<=B){
    result += `${A} `;
    (A%2!==0? A*=2: A+=3);
}
console.log(result);